ValetWatchTest Extension/InterfaceController.m

```diff
@@ -0,0 +1,66 @@
+//
+//  InterfaceController.m
+//  ValetWatchTest Extension
+//
+//  Created by Rodrigo de Souza Reis on 26/10/15.
+//  Copyright © 2015 Square, Inc. All rights reserved.
+//
+
+#import "InterfaceController.h"
+#import <Valet/VALSecureEnclaveValet.h>
+
+@interface InterfaceController()
+
+@property (nonatomic, copy) VALSecureEnclaveValet *secureEnclaveValet;
+@property (nonatomic, copy) NSString *username;
+
+@end
+
+
+@implementation InterfaceController
+
+- (void)awakeWithContext:(id)context {
+    [super awakeWithContext:context];
+
+    // Configure interface objects here.
+}
+
+- (void)willActivate {
+    // This method is called when watch view controller is about to be visible to user
+    [super willActivate];
+    self.secureEnclaveValet = [[VALSecureEnclaveValet alloc] initWithIdentifier:@"UserPresence"];
+    self.username = @"CustomerPresentProof";
+}
+
+- (void)didDeactivate {
+    // This method is called when watch view controller is no longer visible
+    [super didDeactivate];
+}
+
+#pragma mark - Actions
+-(IBAction) setOrUpdateRandomValue:(id)sender {
```

Comment:
Does the SecureEnclave work on Watch? Looks like you're set up to test it here but I'm curious if it works!